### PR TITLE
Code deploy

### PR DESCRIPTION
**Pull request by aninternetof**
Makes travis automatically deploy development commits to beta.rovercode.com and master commits to rovercode.com.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,45 @@
 sudo: true
 before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq build-essential gettext python-dev zlib1g-dev libpq-dev xvfb
-  - sudo apt-get install -qq libtiff4-dev libjpeg8-dev libfreetype6-dev liblcms1-dev libwebp-dev
-  - sudo apt-get install -qq graphviz-dev python-setuptools python3-dev python-virtualenv python-pip
-  - sudo apt-get install -qq firefox automake libtool libreadline6 libreadline6-dev libreadline-dev
-  - sudo apt-get install -qq libsqlite3-dev libxml2 libxml2-dev libssl-dev libbz2-dev wget curl llvm
+- sudo apt-get update -qq
```
Comment (cabarnes) [inline .travis.yml]: The other `.yml` files still have this indentation. Why did you change these?

File: deploy-scripts/remove_prev_install.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+pushd /home/ubuntu/rovercode-web > /dev/null
+docker-compose stop
```
Comment (cabarnes) [inline deploy-scripts/remove_prev_install.sh]: If you run `docker-compose down` it will remove the container as well

File: deploy-scripts/start_app.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/ubuntu/rovercode-web
```
Comment (cabarnes) [inline deploy-scripts/start_app.sh]: You may want to use `&&` to combine all of these. Each is dependent on the previous step succeeding.

File: deploy-scripts/remove_prev_install.sh (hunk 2 above)
Comment (cabarnes) [inline deploy-scripts/remove_prev_install.sh]: You might want to also want to delete the image because each `docker-compose build` will create a new one.

File: .travis.yml (hunk 1 above)
Comment (aninternetof) [inline .travis.yml]: I used the travis client tool to generate the deploy stuff, and it must have changed those indents. I left if because I figured it knew best.

File: deploy-scripts/remove_prev_install.sh (hunk 2 above)
Comment (aninternetof) [inline deploy-scripts/remove_prev_install.sh]: Good ideas.

File: deploy-scripts/start_app.sh (hunk 3 above)
Comment (aninternetof) [inline deploy-scripts/start_app.sh]: Good idea.

**Comment by aninternetof** [comment]
Ok,  changes done!
